assets/stylesheets/styles.css

```diff
@@ -18,11 +18,7 @@ body {
   font-family: "lato", "Roboto", "monsterat";
 }
 
-/* -----------------Header------------------------- */
-nav {
-  display: flex !important;
-  justify-content: space-between;
-}
+
 header {
   padding: 12px;
   background-color: var(--primary);
```

Comment:
Hey @yunweneric that looks good, why do name `#fff` border? Is that not white? If you have a special shade of white you can give it variables names like, `--primary-white`, `--secondary-white`, `--cream-white` etc especially if used in many places.